src/main/java/lotto/domain/LottoRank.java

```diff
@@ -0,0 +1,42 @@
+package lotto.domain;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public enum LottoRank {
+    RANK_1ST(6, 2000000000),
+    RANK_2ND(5, 1500000),
```

Comment:
```suggestion
    RANK_2ND(5, 1_500_000),
```
언더스코어를 통해 가독성을 높일 수 있습니다 ^_^

Reply:
숫자에 언더스코어를 사용할 수 있었네요...